### PR TITLE
Xmitgcm v0.2.2

### PR DESCRIPTION
**Pull request by raphaeldussin**
Read MITgcm mds binary files into xarray

### CODE REVIEW

File: recipes/xmitgcm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - docrep
+
+test:
+  imports:
+    - xmitgcm
+
+about:
+  home: https://github.com/xgcm/xmitgcm
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/xmitgcm/meta.yaml]: It looks like the `MANIFEST.in` includes the `license_file` now. Add a note as a comment in the recipe mentioning that the next release shouldn't need to manually package the `license_file`

File: recipes/xmitgcm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray
+    - dask
+    - numpy
+    - future
```
Comment (synapticarbors) [inline recipes/xmitgcm/meta.yaml]: grep'ing the source distribution, I don't see `docrep` or `future`. Are they needed?

File: recipes/xmitgcm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray
```
Comment (synapticarbors) [inline recipes/xmitgcm/meta.yaml]: In the `setup.py`, there are version restrictions on `xarray` and `dask`? Should they be included?

```
INSTALL_REQUIRES = ['xarray >= 0.10.1', 'dask >= 0.12']
```

File: recipes/xmitgcm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  home: https://github.com/xgcm/xmitgcm
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: Read MITgcm mds binary files into xarray
+  dev_url: https://github.com/xgcm/xmitgcm
+  doc_url: https://xmitgcm.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - raphaeldussin
```
Comment (synapticarbors) [inline recipes/xmitgcm/meta.yaml]: Have all co-maintainers agreed to be included? If you don't have their approval, please ping them in this Pull Request.

Comment (raphaeldussin) [reply recipes/xmitgcm/meta.yaml]: yes

https://github.com/xgcm/xmitgcm/issues/75

File: recipes/xmitgcm/meta.yaml (hunk 3 above)
Comment (raphaeldussin) [reply recipes/xmitgcm/meta.yaml]: fixed

File: recipes/xmitgcm/meta.yaml (hunk 1 above)
Comment (raphaeldussin) [reply recipes/xmitgcm/meta.yaml]: fixed

File: recipes/xmitgcm/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray >= 0.10.1
+    - dask >= 0.12
```
Comment (jakirkham) [inline recipes/xmitgcm/meta.yaml]: Extra spaces here cause conversion to fail. Fixing with PR ( https://github.com/conda-forge/staged-recipes/pull/6599 ).

**Pull request merged**